extra_tests/snippets/builtin_map.py

```
@@ -24,6 +24,13 @@ def __iter__(self):
 assert next(it) == 2
 assert next(it) == 3
 
+# test for no iterables
+try:
+    map(lambda: 1)
+    assert False, "TypeError expected at map construction"
```

Ruff (0.16.1)

[warning] Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)